pwr.md

```diff
@@ -0,0 +1,144 @@
+# pwr module
+
+This is a python module which allows an application to modify power attributes of a CPU. Manipulation can be done to the frequency of cores, 
+frequnecy of the uncore, frequency profiles can be set to achieve desired performance, as well as that capabilities of a CPU can be obtained and its stats monitered.  
+Configuration is done by first creating core and/or CPU object lists, these objects will store the characteristics of the physical processor.
+Modifying those objects and committing the changes enables the module to take the necessary steps to make those changes on the CPU hardware.
```

Comment:
I don't think you need to go into that many details. Just saying something like "The library will provide a list of core/CPU objects, whose attributes can be changed and committed to the cores/CPUs" or similar.